Gemfile

```diff
@@ -16,15 +16,15 @@ gem 'sass-rails'
 gem 'uglifier'
 gem 'coffee-rails'
 gem 'travis'
+gem 'faker'
+gem 'factory_bot_rails'
```

Comment:
We moved these out of the development test block so they would work in production